docs/articles_en/documentation/openvino-extensibility/custom-openvino-operations.rst

```diff
@@ -9,20 +9,54 @@ Custom OpenVINO Operations
                  custom operations to support models with operations
                  not supported by OpenVINO.
 
-OpenVINO™ Extension API allows you to register custom operations to support models with operations which OpenVINO™ does not support out-of-the-box. This capability requires writing code in C++, so if you are using Python to develop your application you need to build a separate shared library implemented in C++ first and load it in Python using ``add_extension`` API. Please refer to :ref:`Create library with extensions <create_a_library_with_extensions>` for more details on library creation and usage. The remaining part of this document describes how to implement an operation class.
+OpenVINO™ Extension API allows you to register custom operations to support models with operations which OpenVINO™ does not support out-of-the-box. While this process historically involved writing C++ code and constructing a separate shared library to be subsequently loaded into Python using the ``add_extension`` API, it is now also possible to implement custom operations directly within the Python API with some known limitations. Operations implemented in python still won't be loaded into the runtime and may be used for building your own Model. 
```

Comment:
Remove retrospective statements.

Reply:
I do not understand this sentence that is also used in other places: "Operations implemented in python still won't be loaded into the runtime". That's not correct. Or you are trying to describe something that I don't understand.